content/components/selectpanel.mdx

```diff
@@ -9,51 +9,153 @@ import {Box} from '@primer/react'
-  src="https://user-images.githubusercontent.com/40274682/229011161-abc2d548-fcf1-4bed-b5a2-eb88d2121a09.png"
+  src="https://github.com/primer/design/assets/980622/17c26b4f-e4ac-4120-8714-1bfffc2b73da"
 />
-<Caption>Select panel can have as much as the branch selector component, or as little as just a multi-selection list. If the contents request user input beyond a semantic menu(example: the context menu on comments in GitHub), it’s likely you need a select panel.</Caption>
 
-### Additional control slot
+### Trigger
+
+<img width="960" alt="" src="https://github.com/primer/design/assets/980622/f6a1e670-2682-4ce7-8642-24943084c812" />
+
+Select panels can be activated by either a regular [button](/components/button), [icon button](/components/icon-button) or [action menu](/components/action-menu). If the button is intended to represent the current selection, it's crucial to have a label associated with it, either internally (within the button) or externally (adjacent to the button).
+
+### List (single select)
 
 <img
   width="960"
-  alt="Three cropped select panel footers each displaying a different option of a control in the bottom right left corner: Link, button, or checkbox input."
-  src="https://user-images.githubusercontent.com/40274682/229011396-2d2c6132-937b-49dd-974c-1a8d4d0d4737.png"
+  alt="Multi-select"
+  src="https://github.com/primer/design/assets/980622/3d0cc2b5-b958-4b62-805e-f77a9d7bf0d9"
 />
-<Caption>An optional control slot is available in the select panel’s bottom left corner. Avoid use if not absolutely necessary in order to avoid component and decision overwhelm.</Caption>
 
-### Anchoring
+If the select panel doesn't have an additional checkbox in the footer, you can make an instant selection. This means that the select panel will close as soon as you make the selection.
 
-Select panel should be anchored to its trigger. 
+However, if there is an additional form element required for that selection, such as checking an additional option, it is necessary to have a cancel and confirmation button. This prevents the dialog from unexpectedly closing when you click an option, allowing you to review and confirm your selection.
 
-<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
-  <div>
-    <img
-      src="https://user-images.githubusercontent.com/40274682/229016799-9559a4ee-0daf-42d1-b46d-11c64cf182e4.png"
-      alt=""
-    />
-    <Caption>Anchored select panel to a trigger.</Caption>
-  </div>
-</Box>
+To indicate an instant selection, we use checkmarks to show the selected option. For regular single selects, we use visually styled radio buttons. Note they are not semantically radio buttons.
```

Comment:
Updated this and I've tried to avoid difficult terminology. Could you have a read now: 

```
If your selection doesn't trigger a serie of events upon submission and doesn't include a checkbox in the footer, you may consider not adding a cancel/save button. In this scenario, when a selection is made, the select panel will instantly close similar to a menu. This behavior is indicated by a checkmark on the selected item.
For example switching a branch is easily reversible and doesn't pose any risk of sending notifications to other users.

However, if the select panel does cause a serie events upon selection or if the footer includes additional form elements like a checkbox, it becomes necessary to include cancel/save buttons. In such cases, the behavior is indicated visually by radio buttons, which allows users to confirm their choices, before submitting their selections.
For example setting the wrong assignee on an pull request poses the risk of sending notifications to the wrong user therefore the cancel/save button is crucial.
```